Mathlib/Tactic/Linarith/Verification.lean

```diff
@@ -202,7 +202,7 @@ def proveFalseByLinarith (cfg : LinarithConfig) : MVarId → List Expr → MetaM
       let (comps, max_var) ← linearFormsAndMaxVar cfg.transparency inputs
       trace[linarith.detail] "... finished `linearFormsAndMaxVar`."
       trace[linarith.detail] "{comps}"
-      let oracle := cfg.oracle.getD FourierMotzkin.produceCertificate
+      let oracle := cfg.oracle.getD SimplexAlgo.produceCertificate
```

Comment:
Could we have a comment here reminding the reader of the existence of FourierMotzkin?

(For maintainability, I'd like to be sure we can always "fall back" quickly and easily.)

Reply:
I put a comment in the docstring